src/models/ComponentStyle.js

```diff
@@ -60,11 +62,9 @@ export default (
       this.isStatic = !isHRMEnabled && isStaticRules(rules, attrs)
       this.componentId = componentId
 
-      if (!StyleSheet.master.hasInjectedComponent(componentId)) {
```

Comment:
This check is no longer needed?

Reply:
No, it’s performed as part of deferredInject now

Reply:
Sorry, you're right. It's still needed for streaming in some edge cases 😅